src/mrack/data/provisioning-config.yaml

```diff
@@ -50,8 +50,9 @@ virt:
 
 aws:  # aws provider config
     images:
-        # list of ami images key is used in metadata ami image is used for provider e.g.
-        fedora-32: ami-0f1d60751946b66c5  # Fedora-Cloud-Base-32-1.6.x86_64-hvm-eu-central-1-gp2-0
+        # list of ami images based on arch (ami image used in metadata is overriding) e.g.
+        x86_64:
+            fedora-32: ami-0f1d60751946b66c5  # Fedora-Cloud-Base-32-1.6.x86_64-hvm-eu-central-1-gp2-0
```

Comment:
arch is not a required parameter if I got it right, so why not make it usable only by defining a new image name if needed? E.g.: `fedora-32-aarch64`.

Reply:
Yes, this is possible approach and probably easier to maintain. However if we think from other side and metadata being reused for any provider IMO arch should not be bound to the `os` name, in this case fedora-32-aarch64.  This is probably question for @f-trivino  as well.